airflow/hooks/druid_hook.py

```diff
@@ -31,14 +34,18 @@ class DruidHook(BaseHook):
     :type timeout: int
     :param max_ingestion_time: The maximum ingestion time before assuming the job failed
     :type max_ingestion_time: int
+    :param druid_broker_conn_id: The connection id to the Druid broker machine
+    :type druid_broker_conn_id: string
     """
     def __init__(
             self,
             druid_ingest_conn_id='druid_ingest_default',
             timeout=1,
-            max_ingestion_time=None):
+            max_ingestion_time=None,
+            druid_broker_conn_id='druid_broker_default'):
```

Comment:
As discussed, an alternative approach would have been to create a `DruidDBApiHook` (perhaps in this same module) that would derive the `DBApiHook`. But without looking I'm guessing there may be dysfunctional inherited methods as the DBApi support for Druid may not be complete (is that the case ?).

I'd assume most Druid related operators would either use only one or the other hook, rarely both, asking for both `conn_id` may be confusing. I think all hooks ask for just one `conn_id`. Druid is a bit odd from that perspective. Let's think that through.

Reply:
Hey Max @mistercrunch , good point from the user perspective. I am still pretty new to airflow framework as well as the convention. The initial intention for me is to group all the logics(ingestion and querying) into single class and like you says I feel some functions in DBApiHook is not well supported by druid. But I am open to extract out the query part into a separate class and raise exceptions for those not well supported methods from DBApiHook.